ipynb/primary-processing/foo

```diff
@@ -0,0 +1,3 @@
ES_BEESTINGS	AUTOIMMUNE	DIET_RESTRICTIONS_LIST	TONSILS_REMOVED	COUNTRY_OF_BIRTH	NAIL_BITER	ECONOMIC_REGION	ALCOHOL_TYPES_RED_WINE	ENA-CHECKLIST	GLUTEN	WEIGHT_KG	SUBSET_AGE	LinkerPrimerSequence	LIVER_DISEASE	BODY_PRODUCT	MILK_SUBSTITUTE_FREQUENCY	SUBSET_HEALTHY	DEPRESSION_BIPOLAR_SCHIZOPHRENIA	ALLERGIC_TO_UNSPECIFIED	LOWGRAIN_DIET_TYPE	SALTED_SNACKS_FREQUENCY	ALCOHOL_TYPES_UNSPECIFIED	AGE_CAT	ALLERGIC_TO_PEANUTS	SUBSET_BMI	MIGRAINE	PROBIOTIC_FREQUENCY	OLIVE_OIL	AGE_YEARS	EXERCISE_FREQUENCY	HOST_COMMON_NAME	READY_TO_EAT_MEALS_FREQUENCY	ALCOHOL_TYPES_SPIRITSHARD_ALCOHOL	SKIN_CONDITION	BOWEL_MOVEMENT_FREQUENCY	SIBO	FERMENTED_PLANT_FREQUENCY	SUGARY_SWEETS_FREQUENCY	TITLE	DOG	OTHER_CONDTITIONS_LIST	BIRTH_YEAR	EXERCISE_LOCATION	NON_FOOD_ALLERGIES_POISON_IVYOAK	COLLECTION_DATE	OTHER_SUPPLEMENT_FREQUENCY	RACE_OTHER	SUBSET_DIABETES	ANONYMIZED_NAME	PREPARED_MEALS_FREQUENCY	HOMECOOKED_MEALS_FREQUENCY	ALLERGIC_TO_SHELLFISH	SMOKING_FREQUENCY	FRUIT_FREQUENCY	HEIGHT_UNITS	STATE	HAS_PHYSICAL_SPECIMEN	TRAVEL_LOCATIONS_LIST	ENV_FEATURE	WHOLE_GRAIN_FREQUENCY	ENA-BASE-COUNT	NON_FOOD_ALLERGIES_PET_DANDER	VITAMIN_D_SUPPLEMENT_FREQUENCY	ADD_ADHD	DOG_CONTACT	SLEEP_DURATION	PUBLIC	HOST_TAXID	LACTOSE	IBD	COLLECTION_SEASON	CAT	WHOLE_EGGS	BOWEL_MOVEMENT_QUALITY	IBS	TYPES_OF_PLANTS	COLLECTION_TIMESTAMP	DEODORANT_USE	MULTIVITAMIN	ACNE_MEDICATION_OTC	WEIGHT_CHANGE	ALLERGIC_TO_I_HAVE_NO_FOOD_ALLERGIES_THAT_I_KNOW_OF	DIABETES	CDIFF	ALLERGIC_TO_OTHER	HEIGHT_CM	CHICKENPOX	SEASONAL_ALLERGIES	THYROID	ACNE_MEDICATION	ASSIGNED_FROM_GEO	BODY_SITE	ELEVATION	FUNGAL_OVERGROWTH	FLOSSING_FREQUENCY	CSECTION	ONE_LITER_OF_WATER_A_DAY_FREQUENCY	COSMETICS_FREQUENCY	HOST_SUBJECT_ID	LONGITUDE	NON_FOOD_ALLERGIES_SUN	ALCOHOL_TYPES_WHITE_WINE	CARDIOVASCULAR_DISEASE	ALZHEIMERS	CAT_LOCATION	WEIGHT_UNITS	COUNTRY	SAMPLE_TYPE	COMMON_NAME	NON_FOOD_ALLERGIES_DRUG_EG_PENICILLIN	NON_FOOD_ALLERGIES_UNSPECIFIED	FLU_VACCINE_DATE	PKU	SUGAR_SWEETENED_DRINK_FREQUENCY	BMI	ENV_BIOME	CENSUS_REGION	DOG_LOCATION	TEETHBRUSHING_FREQUENCY	ENA-SPOT-COUNT	CONTRACEPTIVE	CONSUME_ANIMAL_PRODUCTS_ABX	LIVINGWITH	RELATIONSHIPS_WITH_OTHERS_IN_STUDY	CLINICAL_CONDITION	ENV_MATTER	FED_AS_INFANT	PHYSICAL_SPECIMEN_REMAINING	LAST_TRAVEL	EPILEPSY_OR_SEIZURE_DISORDER	MILK_CHEESE_FREQUENCY	DNA_EXTRACTED	ALCOHOL_TYPES_BEERCIDER	ROOMMATES	HIGH_FAT_RED_MEAT_FREQUENCY	ALTITUDE	SEX	ALLERGIC_TO_TREE_NUTS	ASD	PREGNANT_DUE_DATE	SOFTENER	LUNG_DISEASE	CAT_CONTACT	ALCOHOL_FREQUENCY	POULTRY_FREQUENCY	FROZEN_DESERT_FREQUENCY	BODY_HABITAT	ROOMMATES_IN_STUDY	APPENDIX_REMOVED	DRINKING_WATER_SOURCE	SEAFOOD_FREQUENCY	BMI_CAT	SURVEY_ID	DIET_TYPE	SUBSET_ANTIBIOTIC_HISTORY	RED_MEAT_FREQUENCY	DEPTH	REQUIRED_SAMPLE_INFO_STATUS	LEVEL_OF_EDUCATION	BREASTMILK_FORMULA_ENSURE	ALCOHOL_CONSUMPTION	Description	BarcodeSequence	MEAT_EGGS_FREQUENCY	RACE	KIDNEY_DISEASE	COLLECTION_TIME	VITAMIN_B_SUPPLEMENT_FREQUENCY	LATITUDE	VEGETABLE_FREQUENCY	SUBSET_IBD	ANTIBIOTIC_HISTORY	POOL_FREQUENCY	PREGNANT	ALCOHOL_TYPES_SOUR_BEERS	SIMPLE_BODY_SITE	TITLE_ACRONYM	TITLE_BODY_SITE	HMP_SITE
```

Comment:
It isn't part of the repo from what I can see
On Dec 10, 2015 2:29 PM, "J W Debelius" <notifications@github.com> wrote:

> In ipynb/primary-processing/foo
> <https://github.com/biocore/American-Gut/pull/182#discussion_r47296490>:
>
> > @@ -0,0 +1,3 @@
> > +#SampleID	original-sample-ids	PHYSICAL_SPECIMEN_LOCATION	LAST_MOVE	NON_FOOD_ALLERGIES_BEESTINGS	AUTOIMMUNE	DIET_RESTRICTIONS_LIST	TONSILS_REMOVED	COUNTRY_OF_BIRTH	NAIL_BITER	ECONOMIC_REGION	ALCOHOL_TYPES_RED_WINE	ENA-CHECKLIST	GLUTEN	WEIGHT_KG	SUBSET_AGE	LinkerPrimerSequence	LIVER_DISEASE	BODY_PRODUCT	MILK_SUBSTITUTE_FREQUENCY	SUBSET_HEALTHY	DEPRESSION_BIPOLAR_SCHIZOPHRENIA	ALLERGIC_TO_UNSPECIFIED	LOWGRAIN_DIET_TYPE	SALTED_SNACKS_FREQUENCY	ALCOHOL_TYPES_UNSPECIFIED	AGE_CAT	ALLERGIC_TO_PEANUTS	SUBSET_BMI	MIGRAINE	PROBIOTIC_FREQUENCY	OLIVE_OIL	AGE_YEARS	EXERCISE_FREQUENCY	HOST_COMMON_NAME	READY_TO_EAT_MEALS_FREQUENCY	ALCOHOL_TYPES_SPIRITSHARD_ALCOHOL	SKIN_CONDITION	BOWEL_MOVEMENT_FREQUENCY	SIBO	FERMENTED_PLANT_FREQUENCY	SUGARY_SWEETS_FREQUENCY	TITLE	DOG	OTHER_CONDTITIONS_LIST	BIRTH_YEAR	EXERCISE_LOCATION	NON_FOOD_ALLERGIES_POISON_IVYOAK	COLLECTION_DATE	OTHER_SUPPLEMENT_FREQUENCY	RACE_OTHER	SUBSET_DIABETES	ANONYMIZED_NAME	PREPARED_MEALS_FREQUENCY	HOMECOOKED_MEALS_FREQUENCY
>  	ALLERGI
>  C_TO_SHELLFISH	SMOKING_FREQUENCY	FRUIT_FREQUENCY	HEIGHT_UNITS	STATE	HAS_PHYSICAL_SPECIMEN	TRAVEL_LOCATIONS_LIST	ENV_FEATURE	WHOLE_GRAIN_FREQUENCY	ENA-BASE-COUNT	NON_FOOD_ALLERGIES_PET_DANDER	VITAMIN_D_SUPPLEMENT_FREQUENCY	ADD_ADHD	DOG_CONTACT	SLEEP_DURATION	PUBLIC	HOST_TAXID	LACTOSE	IBD	COLLECTION_SEASON	CAT	WHOLE_EGGS	BOWEL_MOVEMENT_QUALITY	IBS	TYPES_OF_PLANTS	COLLECTION_TIMESTAMP	DEODORANT_USE	MULTIVITAMIN	ACNE_MEDICATION_OTC	WEIGHT_CHANGE	ALLERGIC_TO_I_HAVE_NO_FOOD_ALLERGIES_THAT_I_KNOW_OF	DIABETES	CDIFF	ALLERGIC_TO_OTHER	HEIGHT_CM	CHICKENPOX	SEASONAL_ALLERGIES	THYROID	ACNE_MEDICATION	ASSIGNED_FROM_GEO	BODY_SITE	ELEVATION	FUNGAL_OVERGROWTH	FLOSSING_FREQUENCY	CSECTION	ONE_LITER_OF_WATER_A_DAY_FREQUENCY	COSMETICS_FREQUENCY	HOST_SUBJECT_ID	LONGITUDE	NON_FOOD_ALLERGIES_SUN	ALCOHOL_TYPES_WHITE_WINE	CARDIOVASCULAR_DISEASE	ALZHEIMERS	CAT_LOCATION	WEIGHT_UNITS	COUNTRY	SAMPLE_TYPE	COMMON_NAME	NON_FOOD_ALLERGIES_DRUG_EG_PENICILLIN	NON_FOOD_ALLERGIES_UNSPECIFIED	FLU_VACCINE_DATE	PKU
>  	SUGAR_S
>  WEETENED_DRINK_FREQUENCY	BMI	ENV_BIOME	CENSUS_REGION	DOG_LOCATION	TEETHBRUSHING_FREQUENCY	ENA-SPOT-COUNT	CONTRACEPTIVE	CONSUME_ANIMAL_PRODUCTS_ABX	LIVINGWITH	RELATIONSHIPS_WITH_OTHERS_IN_STUDY	CLINICAL_CONDITION	ENV_MATTER	FED_AS_INFANT	PHYSICAL_SPECIMEN_REMAINING	LAST_TRAVEL	EPILEPSY_OR_SEIZURE_DISORDER	MILK_CHEESE_FREQUENCY	DNA_EXTRACTED	ALCOHOL_TYPES_BEERCIDER	ROOMMATES	HIGH_FAT_RED_MEAT_FREQUENCY	ALTITUDE	SEX	ALLERGIC_TO_TREE_NUTS	ASD	PREGNANT_DUE_DATE	SOFTENER	LUNG_DISEASE	CAT_CONTACT	ALCOHOL_FREQUENCY	POULTRY_FREQUENCY	FROZEN_DESERT_FREQUENCY	BODY_HABITAT	ROOMMATES_IN_STUDY	APPENDIX_REMOVED	DRINKING_WATER_SOURCE	SEAFOOD_FREQUENCY	BMI_CAT	SURVEY_ID	DIET_TYPE	SUBSET_ANTIBIOTIC_HISTORY	RED_MEAT_FREQUENCY	DEPTH	REQUIRED_SAMPLE_INFO_STATUS	LEVEL_OF_EDUCATION	BREASTMILK_FORMULA_ENSURE	ALCOHOL_CONSUMPTION	Description	BarcodeSequence	MEAT_EGGS_FREQUENCY	RACE	KIDNEY_DISEASE	COLLECTION_TIME	VITAMIN_B_SUPPLEMENT_FREQUENCY	LATITUDE	VEGETABLE_FREQUENCY	SUBSET_IBD	ANTIBIOTIC_HISTORY
>  	POOL_FR
>  EQUENCY	PREGNANT	ALCOHOL_TYPES_SOUR_BEERS	SIMPLE_BODY_SITE	TITLE_ACRONYM	TITLE_BODY_SITE	HMP_SITE
>
> I assumed it was part of the master directory I downloaded. I thought it
> was supposed to be here. Can remove if thats wrong
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/American-Gut/pull/182/files#r47296490>.
>